concepts/default-parameters/about.md

```diff
@@ -8,13 +8,13 @@ func greeting(name: String = "guest", duration: Int = 2) -> String {
 }
 
 greeting(name: "Bobo", duration: 7)
-// => "Welcome, Bobo. Enjoy your 7 night stay."
+// Returns "Welcome, Bobo. Enjoy your 7 night stay."
```

Comment:
The strings concept doc doesn't have "Returns" on every example line of code. It might be good to pick one style and use it consistently.